test/e2e/conftest.py

```diff
@@ -278,7 +278,7 @@ def __exit__(self, exc_type, exc_value, traceback):
         ami_id = os.getenv("AMI_ID")
         subnet_id = os.getenv("SUBNET_ID")
         security_group_id = os.getenv("SECURITY_GROUP_ID")
-        instance_type = os.getenv("WORKER_INSTANCE_TYPE", default="t3.micro")
```

Comment:
What was the performance increase with this change?

Reply:
It used to take around 14:42 minutes on the Codebuild to run the cross-os-e2e-test on linux, but after this change I found that it takes around 10 minutes on my local.